src/vue/vuex-modules/ui/popup-module.js

```diff
@@ -1,12 +1,10 @@
 import Vue from 'vue/dist/vue' // Vue in a runtime + compiler configuration
 import Popup from '@/vue/components/popup.vue'
-import { getLanguageName } from '@/lib/utility/language-names.js'
 
 // TODO: Add a check for required modules
 export default class PopupModule {
   constructor (store, api, options) {
     // TODO: Direct links to a UI controller is a temporary solution for compatibility with older code
-    const uiController = options.uiController
     this.options = Object.assign(PopupModule.optionsDefaults, options)
```

Comment:
can we rename this to `config` or something to distinguish it more clearly from `$options`.

Reply:
Yes, definitely, that's a good idea